AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialCanvas_Atom_BasicTests.py

```diff
@@ -123,7 +141,46 @@ def MaterialCanvas_BasicFunctionalityChecks_AllChecksPass():
             Tests.material_graph_name_is_test1,
             material_graph_name == "test1")
 
-        # 8. Look for errors and asserts.
+        # 8. Create a new material_graph from material_graph_document_ids[0].
+        material_graph = material_canvas_utils.get_graph(material_graph_document_ids[0])
+        material_graph_id = material_canvas_utils.get_graph_id(material_graph_document_ids[0])
+        Report.result(
+            Tests.material_graph_created,
+            material_graph.typename == "AZStd::shared_ptr<Graph>" and material_graph is not None)
+
+        # 9. Create a new world_position_node in memory.
+        world_position_node = material_canvas_utils.create_node_by_name(material_graph, "World Position")
+        Report.result(
+            Tests.world_position_node_created,
+            world_position_node.typename == "AZStd::shared_ptr<Node>" and world_position_node is not None)
+
+        # 10. Add the world_position_node to the material_graph.
+        # This test will be verified when the nodes are connected as if it doesn't exist the connection won't be made.
+        material_canvas_utils.add_node(material_graph_id, world_position_node, math.Vector2(-200.0, 10.0))
+
+        # 11. Create a new standard_pbr_node in memory.
+        standard_pbr_node = material_canvas_utils.create_node_by_name(material_graph, "Standard PBR")
+        Report.result(
+            Tests.standard_pbr_node_created,
+            standard_pbr_node.typename == "AZStd::shared_ptr<Node>" and standard_pbr_node is not None)
+
+        # 12. Add the standard_pbr_node to the material_graph.
+        # This test will be verified when the nodes are connected as if it doesn't exist the connection won't be made.
+        material_canvas_utils.add_node(material_graph_id, standard_pbr_node, math.Vector2(10.0, 220.0))
+
+        # 13. Create outbound_slot for our world_position_node and inbound_slot for our standard_pbr_node.
+        # This test will be verified when the nodes are connected as if it doesn't exist the connection won't be made.
+        outbound_slot = material_canvas_utils.get_graph_model_slot_id('outPosition')
+        inbound_slot = material_canvas_utils.get_graph_model_slot_id('inPositionOffset')
```

Comment:
what are these slots even on?

Reply:
they are just type IDs for slots, the unique ID is the node so i think all slots that return from `outPosition` for instance are all the same type ID for that slot

no collisions occur because they are unique across each node, but the slot ID itself on a given node is the same